modules/principals/src/main/frontend/src/Userboard/userboardStyle.jsx

```diff
@@ -39,6 +39,12 @@ const userboardStyle = theme => ({
     },
     dialogActions: {
       padding: theme.spacing(2, 3)
+    },
+    newPrincipalButtonWrapper: {
+      margin: theme.spacing(1),
+      position: "fixed",
+      bottom: theme.spacing(2),
+      right: theme.spacing(2),
```

Comment:
Now the Fab for creating a new user or group is slightly misaligned compared to the rest.
```suggestion
      right: theme.spacing(4),
```